root/static/styles/layout.less

```diff
@@ -592,6 +592,16 @@ span.ac-mp {
     margin-left: -5px;
 }
 
+span.caa-icon {
+    background-image: data-uri('../images/icons/picture.png');
+    background-repeat: no-repeat;
+    display: inline-block;
+    height: 18px;
+    margin-left: 5px;
+    vertical-align: middle;
```

Comment:
`vertical-align: bottom;` actually looks more middle-aligned to me with respect to the release link, I guess because the `<a>` for the link doesn't also have `vertical-align: middle;`.